docker/oso-rhel7-zagg-client/vendor/README

```diff
@@ -0,0 +1,6 @@
+DO NOT EDIT THESE FILES!!!
+
+This is a place to put vendor libs that are needed int he docker file.  These files are not written by the Openshift team.  These files are needed for operations.
```

Comment:
"int he" should be "in the"

Also, consider changing "vendor libs" to "3rd party vendor libs." I know that's redundant, but we're trying to define what the "vendor" directory means here in case someone doesn't know that pattern.